packages/next/server/config-shared.ts

```diff
@@ -469,20 +462,14 @@ export const defaultConfig: NextConfig = {
       (Number(process.env.CIRCLE_NODE_TOTAL) ||
         (os.cpus() || { length: 1 }).length) - 1
     ),
-    sharedPool: true,
-    plugins: false,
-    profiling: false,
     isrFlushToDisk: true,
     workerThreads: false,
-    pageEnv: false,
     optimizeCss: false,
     nextScriptWorkers: false,
     scrollRestoration: false,
     externalDir: false,
     reactRoot: Number(process.env.NEXT_PRIVATE_REACT_ROOT) > 0,
-    disableOptimizedLoading: false,
```

Comment:
Are we done with this config? I think some users were still using it to test `async` over `defer`